Chummer/sheets/de-de/xz.language.xslt

```diff
@@ -21,18 +23,22 @@
   <xsl:variable name="lang.Amount"    select="'Menge'"/>
   <xsl:variable name="lang.AP"      select="'DK'"/>
   <xsl:variable name="lang.Applicable"  select="'Anwendbare'"/>
+  <xsl:variable name="lang.Apprentice"  select="'Lehrling'"/>
   <xsl:variable name="lang.AR"      select="'AR'"/>
   <xsl:variable name="lang.Archetype"    select="'Archetyp'"/>
   <xsl:variable name="lang.Area"      select="'Fläche'"/>
   <xsl:variable name="lang.Armor"      select="'Panzerung'"/>
   <xsl:variable name="lang.Arts"      select="'Künste'"/>
   <xsl:variable name="lang.as"      select="'als'"/>
+  <xsl:variable name="lang.ASDF"    select="'A/S/D/F'"/>
   <xsl:variable name="lang.Astral"    select="'Astral'"/>
   <xsl:variable name="lang.Attack"    select="'Angriff'"/>
   <xsl:variable name="lang.ATT"      select="'ANG'"/>
   <xsl:variable name="lang.Attribute"    select="'Attribut'"/>
   <xsl:variable name="lang.Attributes"  select="'Attribute'"/>
   <xsl:variable name="lang.Available"    select="'Verfügbar'"/>
+  <xsl:variable name="lang.Awakened"    select="'Erwachen'"/>
```

Comment:
correct: "'erwacht'"

Reply:
Depending on the usage, it must start with an upper case letter -> 'Erwacht'

Reply:
you are right. "Erwacht" would be right